src/codec/ValueCodec.ts

```diff
@@ -23,6 +25,18 @@ export const ValueCodec: Codec<PbValue, Value> = {
             return {valueType: 'contractId', contractId: value.getContractId()};
         } else if (value.hasDate()) {
             return {valueType: 'date', date: '' + value.getDate()};
+        } else if (value.hasEnum()) {
+            const damlEnum = value.getEnum()!;
+            value.getEnum()!.getEnumId();
```

Comment:
Is this a leftover or sideffecting somehow differently?

Reply:
Leftover, thanks.

Reply:
Addressed by https://github.com/digital-asset/daml-js/pull/101/commits/548c05e8756b740dfe2ef08551138b9571f45ffe